benefit/data/sheets/CaseEvent/CaseEvent-sandl-nonprod.json

```diff
@@ -9,5 +9,16 @@
     "PreConditionState(s)": "notListable;withDwp;responseReceived",
     "PostConditionState": "*",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/07/2022",
+    "CaseTypeID": "Benefit",
+    "ID": "addCaseOutcome",
+    "Name": "Add case outcome",
+    "Description": "Add a case outcome",
+    "DisplayOrder": 14,
+    "PreConditionState(s)": "*",
```

Comment:
I just realised: we might need clarification on what state qualifies this event, and what state the case should be in afterwards (if any specifically)

Reply:
I checked this with Makeda, the case could be any state to be able to record the outcome.